tree.js

```diff
@@ -48,7 +48,7 @@ export class Tree {
 
     if (!tmp.done) {
       this.currentNode = tmp.value;
-      this.ptree();
+      //this.printTree();
```

Comment:
Assuming you left these commented incase we wanted to debug? If so, prehaps we have a debug toggle on the renderer or allow a passthrough parameter that could enable these print statements?

Reply:
Exactly, and I like that call. I'll add a logger class in an upcoming PR that gives us more control of when/where we see logs.